ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/ros2/ROS2BehaviorTreeState.java

```diff
@@ -60,6 +59,10 @@ public void updateSubscription()
    public void updatePublication()
    {
       behaviorTreePublisher.publish();
+
+      // We increment the CRDT update number once per publication,
+      // which done at the SYNC_FREQUENCY.
+      behaviorTreeState.getCRDTInfo().startNextUpdate();
```

Comment:
increment the CRDT update number on publishing instead of subscribing. Makes the number 0 for the first run and publishing happens at a constant rate. it was at a constant rate before but you had to wonder if it was in the callback